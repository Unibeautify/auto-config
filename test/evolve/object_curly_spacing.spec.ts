import { testGenetic } from "../testGenetic";
import prettier from "@unibeautify/beautifier-prettier";
import jsBeautify from "@unibeautify/beautifier-js-beautify";

testGenetic("object_curly_spacing", {
  beautifiers: [prettier, jsBeautify],
  language: "JavaScript",
  originalText: `var obj = { foo: "bar" };`,
  desiredText: `var obj = {foo: "bar"};\n`,
});
