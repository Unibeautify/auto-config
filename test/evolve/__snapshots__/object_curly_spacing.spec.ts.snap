// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`object_curly_spacing: beautifiedText 1`] = `
var obj = {foo: "bar"};

`;

exports[`object_curly_spacing: beautifiers 1`] = `
Array [
  "Prettier",
]
`;

exports[`object_curly_spacing: trimmedOptions 1`] = `
Object {
  "object_curly_spacing": false,
  "quotes": "none",
}
`;
